privacyidea/lib/token.py

```diff
@@ -2027,7 +2027,7 @@ def check_token_list(tokenobject_list, passw, user=None, options=None):
     # add the user to the options, so that every token, that get passed the
     # options can see the user
     options = options or {}
-    options = dict(options.items() + {'user': user}.items())
+    options.update({'user': user})
```

Comment:
Might be a bit nitpicking :-) But if the caller passes ``options``, this change will modify the object in-place, which was not the case before. I think it would be safer to have something like
```python
if options:
    options = options.copy()
else:
    options = {}
options.update(...)
```

Reply:
oups I was not aware of the copy. I think I changed this also in my PR. But you are right.

Reply:
No nitpicking, will fix.

Reply:
Done

Reply:
:+1: